client/src/components/mapping/FhirMappingPanel/FhirResourceTree/secondaryLabel.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Attribute } from '@arkhn/fhir.ts';
+import { useSelector } from 'react-redux';
+import { Icon } from '@blueprintjs/core';
+
+import { IAttribute, IReduxStore } from 'types';
+
+interface SecondaryLabelProps {
+  attribute: Attribute;
+}
+
+export const SecondaryLabel = ({
+  attribute
+}: SecondaryLabelProps): React.ReactElement => {
+  const attributesForResource: { [k: string]: IAttribute } = useSelector(
+    (state: IReduxStore) => state.resourceInputs.attributesMap
+  );
+  const attributesWithInputs = Object.keys(attributesForResource).filter(
+    path => attributesForResource[path].inputs.length > 0
+  );
+
+  const checkHasChildInputs = (pathString: string) =>
+    attributesWithInputs.some(
+      el => el !== pathString && el.startsWith(pathString)
+    );
+
+  const checkHasInputs = (pathString: string) =>
```

Comment:
Maybe `hasChildInputs` and `hasInputs` would be better than `checkHasChildInputs` and `checkHasInputs`

Reply:
we already have variables with these names, which are initialised using these functions.